src/components/Home.jsx

```diff
@@ -47,7 +47,7 @@ const Home = ({ setActors, actors, scriptOption }) => {
       <ul id='actorList'>
         <h3>Actors</h3>
         {actors.map((actor) => (
-          <li key={`${actor}`}>{actor[0] + ' ' + actor[1]}</li>
+          <li key={actor}>{actor[0] + ' ' + actor[1]}</li>
```

Comment:
Were all the other changes in this commit intended? The message only references this line's change.

Reply:
I don't know how those all got in there. I should have double checked, thank you for catching that. I'll fix it.